app/views/users/two_factor_authentication_setup/index.html.slim

```diff
@@ -11,25 +11,21 @@ p.mt-tiny.mb0#2fa-description
   = f.label :phone, class: 'block'
     strong.left = t('devise.two_factor_authentication.otp_phone_label')
     span.ml1.italic = t('devise.two_factor_authentication.otp_phone_label_info')
-    = f.input :phone, as: :tel, label: false, required: true,
-        input_html: { 'aria-describedby' => '2fa-description', class: 'phone' }
+  = f.input :phone, as: :tel, label: false, required: true,
+      input_html: { 'aria-describedby' => '2fa-description', class: 'phone sm-col-8 mb4' }
   .mb3
-    = label_tag 'two_factor_setup_form[otp_method]',
-      t('devise.two_factor_authentication.otp_method.title'),
-      class: 'block bold'
-    fieldset.border-none
-      legend.hide = t('devise.two_factor_authentication.otp_method.instruction')
-      label.radio.mb2.block
-        = radio_button_tag 'two_factor_setup_form[otp_method]', :sms,
-          checked: true
-        span.indicator#sms-otp-description
-        = t('devise.two_factor_authentication.otp_method.sms')
-      label.radio.mb0.block
-        = radio_button_tag 'two_factor_setup_form[otp_method]', :voice
-        span.indicator#voice-otp-description
-        = t('devise.two_factor_authentication.otp_method.voice')
-    = label_tag 'two_factor_setup_form[otp_method]',
-      t('devise.two_factor_authentication.otp_method.instruction'),
-      class: 'block'
+    h2.h4 = t('devise.two_factor_authentication.otp_method.title')
```

Comment:
It's normal.  Basically we are saying that we want a heading that is semantically an h2 but appears as an h4.